Ginger/Ginger/Activities/ActionsPOMListViewHelper.cs

```diff
@@ -0,0 +1,364 @@
+#region License
+/*
+Copyright © 2014-2021 European Support Limited
+
+Licensed under the Apache License, Version 2.0 (the "License")
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at 
+
+http://www.apache.org/licenses/LICENSE-2.0 
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS, 
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+See the License for the specific language governing permissions and 
+limitations under the License. 
+*/
+#endregion
+
+using amdocs.ginger.GingerCoreNET;
+using Amdocs.Ginger.Common;
+using Amdocs.Ginger.Common.UIElement;
+using Amdocs.Ginger.Repository;
+using Amdocs.Ginger.UserControls;
+using Ginger.Activities;
+using Ginger.Repository;
+using Ginger.UserControlsLib.UCListView;
+using GingerCore;
+using GingerCore.Actions;
+using GingerCore.Activities;
+using System.Collections.Generic;
+using System.Linq;
+using System.Windows;
+
+namespace Ginger.BusinessFlowPages.ListHelpers
+{
+    public class ActionsPOMListViewHelper : IListViewHelper
+    {
+        Act mAct;
+        public Context mContext;
+        private Agent mAgent;
+        public General.eRIPageViewMode PageViewMode { get; set; }
+
+        UcListView mListView = null;
+        public UcListView ListView
+        {
+            get
+            {
+                return mListView;
+            }
+            set
+            {
+                if (mListView != value)
+                {
+                    //if (mListView != null)
+                    //{
+                    //    mListView.UcListViewEvent -= ListView_UcListViewEvent;
+                    //}
+                    mListView = value;
+                    //if (mListView != null)
+                    //{
+                    //    mListView.UcListViewEvent += ListView_UcListViewEvent;
+                    //}
+                }
+            }
+        }
+
+        public bool AllowExpandItems { get; set; } = true;
+
+        public bool ExpandItemOnLoad { get; set; } = false;
+
+        public ActionsPOMListViewHelper(Context context, General.eRIPageViewMode pageViewMode)
```

Comment:
I see we already have pageViewMode been used, so I think it will be better to merge all ActionsViewhelp into one class...